node_modules/oae-principals/lib/definitiveDeletion.js

```diff
@@ -0,0 +1,1851 @@
+/*!
+ * Copyright 2017 Apereo Foundation (AF) Licensed under the
+ * Educational Community License, Version 2.0 (the 'License'); you may
+ * not use this file except in compliance with the License. You may
+ * obtain a copy of the License at
+ *
+ *     http://opensource.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an 'AS IS'
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+var _ = require('underscore');
+var fs = require('fs');
+var util = require('util');
+
+var AuthzAPI = require('oae-authz');
+var AuthzConstants = require('oae-authz/lib/constants').AuthzConstants;
+var AuthenticationConstants = require('oae-authentication/lib/constants').AuthenticationConstants;
+var Cassandra = require('oae-util/lib/cassandra');
+var ConfigAPI = require('oae-config');
+var ContentAPI = require('oae-content');
+var ContentConfig = require('oae-config').config('oae-content');
+var ContentConstants = require('oae-content/lib/constants').ContentConstants;
+var ContentUtil = require('oae-content/lib/internal/util');
+var Context = require('oae-context').Context;
+var DiscussionAPI = require('oae-discussions/lib/api.discussions');
+var DiscussionsDAO = require('oae-discussions/lib/internal/dao');
+var EmailAPI = require('oae-email');
+var FolderAPI = require('oae-folders');
+var FolderLibraryDAO = require('oae-folders/lib/internal/foldersLibrary');
+var FoldersConstants = require('oae-folders/lib/constants').FoldersConstants;
+var FoldersDAO = require('oae-folders/lib/internal/dao');
+var FollowingAPI = require('oae-following');
+var FollowingConstants = require('oae-following/lib/constants').FollowingConstants;
+var FollowingDAO = require('oae-following/lib/internal/dao');
+var GroupAPI = require('./api.group');
+var LoginId = require('oae-authentication/lib/model').LoginId;
+var MeetingsAPI = require('oae-jitsi');
```

Comment:
When I don't put the path `oae-jitsi/lib/api.meetings` oae don't find the methods I call ... 

I have the same problem on `oae-folders/lib/api.js`line 33/34 with `var PrincipalsAPI = require('oae-principals');`, I have to specified `var PrincipalsAPI = require('oae-principals/lib/api.group');` and other same problem in `oae-principals/lib/internal/dao.js` line 317 when I use `require('oae-authentication').getUserLoginIds(...` it's because when I try `const AuthenticationAPI = require('oae-authentication');` and calling the method with `AuthenticationAPI.getUserLoginIds(...` it doesn't work ... 

I'm probably do something wrong ^^'

Reply:
Have you tried with `var MeetingsAPI = require('oae-jitsi').Meetings;`?